.github/workflows/build.yml

```diff
@@ -126,6 +126,8 @@ jobs:
             mkdir -p "${{runner.workspace}}/pkg/usr/share/icons/hicolor/${LOGO_SIZE}x${LOGO_SIZE}/"
             cp "./psst-gui/assets/${LOGO}" "$_/psst.png"
           done
+          mkdir -p "${{runner.workspace}}/pkg/usr/share/icons/hicolor/scalable/apps/"
+          cp "./psst-gui/assets/psst.svg" $_
```

Comment:
Could you call the SVG `logo.svg` to match the other assets included?

Reply:
Done